test/openshift/e2e/sequential/1-077_validate_workload_status_monitoring_alert/02-verify-alert.yaml

```diff
@@ -1,13 +1,13 @@
-# apiVersion: kuttl.dev/v1beta1
-# kind: TestStep
-# commands:
-# - script: sleep 10m
-# - script: |
-#     jq '.data.alerts' <<< "$(oc -n openshift-monitoring exec -c prometheus prometheus-k8s-0 -- curl -s   'http://localhost:9090/api/v1/alerts')" > alerts.json
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: sleep 10m
```

Comment:
**sleep 10m** would wait for 10m even when the condition is met and delay other test executions.Can we have a polling mechanism to avoid this?